x-pack/legacy/plugins/graph/public/components/guidance_panel/guidance_panel.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { ReactNode } from 'react';
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiLink } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import classNames from 'classnames';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export interface GuidancePanelProps {
+  onFillWorkspace: () => void;
+  onOpenFieldPicker: () => void;
+  onOpenDatasourcePicker: () => void;
+  hasDatasource: boolean;
+  hasFields: boolean;
+}
+
+function ListItem({
+  children,
+  disabled,
+  selected,
+}: {
+  children: ReactNode;
+  disabled: boolean;
+  selected: boolean;
+}) {
+  return (
+    <li
+      className={classNames('gphGuidancePanel__item', {
+        'gphGuidancePanel__item--disabled': disabled,
+      })}
+      aria-disabled={disabled}
+    >
+      {selected && (
+        <EuiIcon
+          color="secondary"
+          type="check"
+          className="gphGuidancePanel__itemIcon"
+          aria-hidden={true}
+          size="l"
+        />
+      )}
+      {children}
+    </li>
+  );
+}
+
+export function GuidancePanel(props: GuidancePanelProps) {
+  const {
+    onFillWorkspace,
+    onOpenFieldPicker,
+    onOpenDatasourcePicker,
+    hasDatasource,
+    hasFields,
+  } = props;
+
+  return (
+    <EuiFlexGroup justifyContent="center">
+      <EuiFlexItem className="gphGuidancePanel">
+        <EuiPanel>
+          <EuiFlexGroup direction="column" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="graphApp" size="xxl" />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <h1>
```

Comment:
Can we add an `aria-label` here? "Let's get started!" doesn't communicate much context.

Maybe something like "Let's get started building your first graph!" 

By also staying close to what the visual copy is, I'm hoping to also minimize confusion for sighted users using screen readers.